--- PHPMailer-5.2.7/class.phpmailer.php~	2013-09-12 10:52:38.000000000 +0300
+++ PHPMailer-5.2.7/class.phpmailer.php	2013-11-09 13:09:34.941928857 +0200
@@ -185,7 +185,7 @@
      * @type string
      * @deprecated Should not be needed now there is an autoloader.
      */
-    public $PluginDir = '';
+    public $PluginDir = '/usr/share/php/phpmail/';
 
     /**
      * The email address that a reading confirmation should be sent to.
@@ -1323,7 +1323,7 @@
      * @return bool
      * @access public
      */
-    public function setLanguage($langcode = 'en', $lang_path = 'language/')
+    public function setLanguage($langcode = 'en', $lang_path = '/usr/share/php/phpmailer/language/') {
     {
         //Define full set of translatable strings
         $PHPMAILER_LANG = array(
