--- PHPMailer-5.2.5/class.phpmailer.php~	2013-05-24 09:51:19.087751571 +0300
+++ PHPMailer-5.2.5/class.phpmailer.php	2013-05-24 09:52:08.203626736 +0300
@@ -134,7 +134,7 @@
    * is in a different directory than the PHP include path.
    * @var string
    */
-  public $PluginDir         = '';
+  public $PluginDir         = '/usr/share/php/phpmail/';
 
   /**
    * Holds PHPMailer version.
@@ -1134,7 +1134,7 @@
    * @return bool
    * @access public
    */
-  function SetLanguage($langcode = 'en', $lang_path = 'language/') {
+  function SetLanguage($langcode = 'en', $lang_path = '/usr/share/php/phpmailer/language/') {
     //Define full set of translatable strings
     $PHPMAILER_LANG = array(
       'authenticate'         => 'SMTP Error: Could not authenticate.',
