--- phpMailer_v2.3/class.phpmailer.php~	2010-01-12 16:05:45.000000000 +0200
+++ phpMailer_v2.3/class.phpmailer.php	2010-01-12 16:05:46.997539898 +0200
@@ -134,7 +134,7 @@
    * is in a different directory than the PHP include path.
    * @var string
    */
-  public $PluginDir         = '';
+  public $PluginDir         = '/usr/share/php/phpmail/';
 
   /**
    * Holds PHPMailer version.
@@ -663,7 +663,7 @@
    * @access public
    * @return bool
    */
-  function SetLanguage($lang_type = 'en', $lang_path = 'language/') {
+  function SetLanguage($lang_type = 'en', $lang_path = '/usr/share/php/phpmailer/language/') {
     if( !(@include $lang_path.'phpmailer.lang-'.$lang_type.'.php') ) {
       $PHPMAILER_LANG = array();
       $PHPMAILER_LANG["provide_address"]      = 'You must provide at least one ' .
