--- PHPMailer_5.2.1/test/phpmailerTest.php~	2012-04-22 14:55:00.201857103 +0300
+++ PHPMailer_5.2.1/test/phpmailerTest.php	2012-04-22 14:54:00.232668489 +0300
@@ -13,7 +13,6 @@
 * @license http://www.gnu.org/copyleft/lesser.html GNU Lesser General Public License
 */
 
-require 'PHPUnit/Framework.php';
 
 $INCLUDE_DIR = "../";
 
@@ -146,7 +145,7 @@
         $ReportBody .= "---------------------" . $eol;
         $ReportBody .= "Unit Test Information" . $eol;
         $ReportBody .= "---------------------" . $eol;
-        $ReportBody .= "phpmailer version: " . PHPMailer::VERSION . $eol;
+        $ReportBody .= "phpmailer version: " . $this->Mail->Version . $eol;
         $ReportBody .= "Content Type: " . $this->Mail->ContentType . $eol;
         
         if(strlen($this->Mail->Host) > 0)
@@ -637,7 +636,6 @@
 	    $this->Mail->CreateHeader();
 	    $this->assertFalse($this->Mail->set('x', 'y'), 'Invalid property set succeeded');
 	    $this->assertTrue($this->Mail->set('Timeout', 11), 'Valid property set failed');
-	    $this->Mail->getFile(__FILE__);
 	}
 }  
  
